src/main/java/models/Client.java

```diff
@@ -0,0 +1,15 @@
+package models;
+
+public class Client extends User{
```

Comment:
check item 5 in task and add missing method

Reply:
I have this method "uniqueAbility". I extends it from Class User and then @Override it.

Reply:
the same, see previous comment

Reply:
+